kokoro/ubuntu/release.sh

```diff
@@ -8,14 +8,16 @@ set -o errexit
 # Display commands being run.
 set -o xtrace
 
-# More recent Cloud SDK requires Python 3.5 (b/194714889)
-export CLOUDSDK_PYTHON=python3.5
```

Comment:
I'm confused. You added this yesterday, and it did seem to fix Kokoro. Why did it work then?

Reply:
I think that was Kokoro for Windows which likely has a more up-to-date Cloud SDK installed?

Reply:
Aha!

Reply:
Well, I guess Windows uses `.bat`, but whatever. As long as this fixes all problems, we are good.